src/pages/iou/steps/NewRequestAmountPage.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import _ from 'underscore';
+import ONYXKEYS from '../../../ONYXKEYS';
+import Navigation from '../../../libs/Navigation/Navigation';
+import ROUTES from '../../../ROUTES';
+import compose from '../../../libs/compose';
+import * as ReportUtils from '../../../libs/ReportUtils';
+import * as CurrencyUtils from '../../../libs/CurrencyUtils';
+import CONST from '../../../CONST';
+import reportPropTypes from '../../reportPropTypes';
+import * as IOU from '../../../libs/actions/IOU';
+import useLocalize from '../../../hooks/useLocalize';
+import withCurrentUserPersonalDetails, {withCurrentUserPersonalDetailsDefaultProps, withCurrentUserPersonalDetailsPropTypes} from '../../../components/withCurrentUserPersonalDetails';
+import MoneyRequestAmountForm from './MoneyRequestAmountForm';
+import * as IOUUtils from '../../../libs/IOUUtils';
+import FullPageNotFoundView from '../../../components/BlockingViews/FullPageNotFoundView';
+
+const propTypes = {
+    route: PropTypes.shape({
+        params: PropTypes.shape({
+            iouType: PropTypes.string,
+            reportID: PropTypes.string,
+        }),
+    }),
+
+    /** The report on which the request is initiated on */
+    report: reportPropTypes,
+
+    /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
+    iou: PropTypes.shape({
+        id: PropTypes.string,
+        amount: PropTypes.number,
+        currency: PropTypes.string,
+        participants: PropTypes.arrayOf(
+            PropTypes.shape({
+                accountID: PropTypes.number,
+                login: PropTypes.string,
+                isPolicyExpenseChat: PropTypes.bool,
+                isOwnPolicyExpenseChat: PropTypes.bool,
+                selected: PropTypes.bool,
+            }),
+        ),
+    }),
+
+    ...withCurrentUserPersonalDetailsPropTypes,
+};
+
+const defaultProps = {
+    route: {
+        params: {
+            iouType: '',
+            reportID: '',
+        },
+    },
+    report: {},
+    iou: {
+        id: '',
+        amount: 0,
+        currency: CONST.CURRENCY.USD,
+        participants: [],
+    },
+    ...withCurrentUserPersonalDetailsDefaultProps,
+};
+
+function NewRequestAmountPage(props) {
```

Comment:
We should use prop destructuring -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring